web-server/package.json

```diff
@@ -1,77 +1,50 @@
 {
-  "name": "hurado-server",
+  "name": "backend",
```

Comment:
Not a fan of this. One day you'll have two projects. And they'll both have `backend` as the name of the package.

Or one day you're going to have one project with two large independent sections. And again, they'll both have "backend" as the name of the thing.